<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 227, 246);
font: 75 italic bold 11pt &quot;Liberation Serif&quot;;
color:rgb(26, 14, 165)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="gFreqTime" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>540</width>
      <height>420</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 italic 30pt &quot;Liberation Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Cosmic Ray Telescope Monitor</string>
    </property>
   </widget>
   <widget class="QFrame" name="wifi_frame">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>51</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{ 
       background-color: transparent;
         background-image: url(&quot;wifi_off.png&quot;);
         background-repeat: none; border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>94</width>
      <height>84</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QFrame
{
    background-color: transparent;
    background-image: url(&quot;utfsm_silab.png&quot;);
    background-repeat: none;    
    border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="gateButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>127</width>
      <height>127</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: transparent;
    background-image: url(&quot;default_button.png&quot;);
    background-repeat: none;    
    border: none;
}
QPushButton:hover
{
   background-image: url(&quot;hover_button.png&quot;);
}

QPushButton:pressed
{
   background-image: url(&quot;press_button.png&quot;);
}
	border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Set Gate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="npointsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>127</width>
      <height>127</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: transparent;
    background-image: url(&quot;default_button.png&quot;);
    background-repeat: none;    
    border: none;
}
QPushButton:hover
{
   background-image: url(&quot;hover_button.png&quot;);
}

QPushButton:pressed
{
   background-image: url(&quot;press_button.png&quot;);
}
	border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Set Plots 
 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="azimuthButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>127</width>
      <height>127</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: transparent;
    background-image: url(&quot;default_button.png&quot;);
    background-repeat: none;    
    border: none;
}
QPushButton:hover
{
   background-image: url(&quot;hover_button.png&quot;);
}

QPushButton:pressed
{
   background-image: url(&quot;press_button.png&quot;);
}
	border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Set Azimuth 
 Angle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polarButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>127</width>
      <height>127</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: transparent;
    background-image: url(&quot;default_button.png&quot;);
    background-repeat: none;    
    border: none;
}
QPushButton:hover
{
   background-image: url(&quot;hover_button.png&quot;);
}

QPushButton:pressed
{
   background-image: url(&quot;press_button.png&quot;);
}
	border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Set Polar 
 Angle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="distanceButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>400</y>
      <width>127</width>
      <height>127</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color: transparent;
    background-image: url(&quot;default_button.png&quot;);
    background-repeat: none;    
    border: none;
}
QPushButton:hover
{
   background-image: url(&quot;hover_button.png&quot;);
}

QPushButton:pressed
{
   background-image: url(&quot;press_button.png&quot;);
}
	border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Set 
 Dstance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial enable</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_avg">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>[Hz]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Stantard deviation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_sd">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>[Hz]</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>560</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>570</y>
      <width>26</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>580</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
